tests/testutil/testutil.go

```diff
@@ -111,3 +113,22 @@ func GetHostname(URL string) string {
 	}
 	return u.Hostname()
 }
+
+// stringInSlice returns true if string a value is equals to any element of the slice otherwise false
+func stringInSlice(a string, list []string) bool {
+	for _, b := range list {
```

Comment:
nitpick, but can we rename `b` to `i` or `l`, that way it's somewhat relevant inside the loop.